Add release highlights for 1.4.0 (#4135)
elastic/cloud-on-k8s: docs/release-notes/highlights-1.4.0.asciidoc

[[release-highlights-1.4.0]]
== 1.4.0 release highlights

[float]
[id="{p}-140-new-and-notable"]
=== New and notable

New and notable changes in version 1.4.0 of {n}. See <<release-notes-1.4.0>> for the full list of changes.

[float]
[id="{p}-140-agent-support"]
==== Support for Elastic Agent

> link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] provides a unified way to monitor logs, metrics, and other types of data from your Kubernetes infrastructure quickly and easily.  A single agent deployment can be used to replace multiple Beats deployments that were previously necessary to collect the different types of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.

Reviewer: To be consistent: 

```suggestion
link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] provides a unified way to monitor logs, metrics, and other types of data from your Kubernetes infrastructure quickly and easily. A single Elastic Agent deployment can be used to replace multiple Beats deployments that were previously necessary to collect the different types of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.
```

Also, shouldn't the first link be https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation-configuration.html? The fleet-overview one talks mostly about Fleet and central management which is mostly not there for standalone mode.

> link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] is a single, unified way to add monitoring for logs, metrics, and other types of data to each Kubernetes node. With a single agent, it's easier and faster to deploy monitoring across your infrastructure; also there is no need to deploy multiple instances of Beats for different kinds of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.

Reviewer 2: I know the description comes from the product page. It just feels like a weird way of explaining things. Feel free to ignore my suggestion though.
```suggestion
link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] provides a unified way to monitor logs, metrics, and other types of data from your Kubernetes infrastructure quickly and easily.  A single agent deployment can be used to replace multiple Beats deployments that were previously necessary to collect the different types of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.
```

> link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] is a single, unified way to add monitoring for logs, metrics, and other types of data to each Kubernetes node. A single agent makes it easier and faster to deploy monitoring across your infrastructure. There is no need with agent to deploy multiple instances of Beats for different kinds of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.

Reviewer 3: ```suggestion
link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] is a single, unified way to add monitoring for logs, metrics, and other types of data to each Kubernetes node. With a single agent, it's easier and faster to deploy monitoring across your infrastructure; also there is no need to deploy multiple instances of Beats for different kinds of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.
```
